### DISCUSSION TITLE
I got the confirmation email but I can't use it

### DISCUSSION THREAD

**Discussion by zTrolly** [community/community#18992] [Copilot Conversations] [answered]
I received an email confirming that my account now has access to copilot but when I start typing in vsCode the extension denies my access
```
[INFO] 
[ghostText] [2022-06-20T17:36:34.010Z] Cancelled after extractPrompt
[INFO] [default] [2022-06-20T17:36:34.237Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-20T17:36:34.826Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 589 ms
[INFO] [fetch] [2022-06-20T17:36:34.835Z] Please upgrade your Copilot extension to continue using this service.
```

<img width="560" alt="Captura de Tela 2022-06-18 às 23 13 48" src="https://user-images.githubusercontent.com/61760048/174654231-61fc3ca2-4501-463b-8008-fd20a2590526.png">

**Comment by charleprr** [comment, marked as answer]
Updating your VSCode to [the latest version](https://code.visualstudio.com/download) should fix the issue.

If this does not work, make sure you are using the latest version of [the GitHub Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and try signing in again with your GitHub account.
